staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -236,7 +236,7 @@ func (r *crdHandler) serveResource(w http.ResponseWriter, req *http.Request, req
 	case "update":
 		return handlers.UpdateResource(storage, requestScope, discovery.NewUnstructuredObjectTyper(nil), r.admission)
 	case "patch":
-		return handlers.PatchResource(storage, requestScope, r.admission, unstructured.UnstructuredObjectConverter{}, supportedTypes)
```

Comment:
Not sure how these tests were working previously, clearly the schema reference object couldn't have been constructed correctly.

Reply:
schemaReferenceObject is only used for SMP, which CRD doesn't support. it would have ended up an unstructured object in the CRD patch handler (New() returns unstructured, and the unstructured convertor keeps it unstructured).